src/styles/utils/_variables.scss

```
@@ -0,0 +1,8 @@
+$weekdays: "mon" 1, "tue" 2, "wen" 3, "thu" 4, "fri" 5, "sat" 6, "sun" 7;
```

If we have a long variable, it is better to write it like so:
```suggestion
$weekdays: 
  "mon" 1, 
  "tue" 2,
  "wen" 3, 
  "thu" 4, 
  "fri" 5,
  "sat" 6,
  "sun" 7;
```